src/services/services.ts

```diff
@@ -2101,7 +2101,7 @@ namespace ts {
         };
     }
 
-    export function preProcessFile(sourceText: string, readImportFiles = true): PreProcessedFileInfo {
+    export function preProcessFile(sourceText: string, readImportFiles = true, isJavaScript = false): PreProcessedFileInfo {
         let referencedFiles: FileReference[] = [];

```

Comment:
since `define` calls should also be counted then name should be smth like `includeDepencendiesFromRequireOrDefine` which is a bit too long. I'll stick with `detectJavaScriptModules` for now